### DISCUSSION TITLE
Random 500 error on search with API

### DISCUSSION THREAD

**Discussion by QrabblApp** [supabase/supabase#21727] [Questions] [answered]
Hello,

I'm trying to search users with the API like:
https://xxxxxxxx.supabase.co/rest/v1/users?or=(first_name.ilike.%[SearchString]%)

When I use a string that starts with 'ba', 'ca' or 'be' I'm getting 500 errors. 
While I'm not getting any errors on other search strings, even when the result is empty.

The funny thing is, this explainer video has the same issue at 8:01: https://youtu.be/jMLjyYwqJN0?feature=shared&t=481

**Comment by GaryAustin1** [comment, marked as answer]
Probably you need to encode % for use in a URL. Note this https://postgrest.org/en/v12/references/api/tables_views.html#operators says you can use * in place of % to avoid that.

**Reply by QrabblApp** [reply]
Thanks! Replacing '%' with '*' worked for me.
